.circleci/config.yml

```diff
@@ -132,3 +210,19 @@ workflows:
             branches:
               only:
                - master
+  # Runs on every commit without triggers
+  commit:
+    jobs:
+    - smoke-test
```

Comment:
Can we run these after the build, but before the deploys instead? i.e. there's no point running the smoke tests if the build fails, and we shouldn't deploy if the smoke tests fail.

Reply:
Will do.